src/components/capture/GenerateButton.tsx

```diff
@@ -78,21 +80,26 @@ function CaptureGenerateButton({ disabled, callFailed, subscription }: Props) {
         useEndpointConfigStore_encryptedEndpointConfig_data();
     const endpointConfigErrorsExist = useEndpointConfigStore_errorsExist();
 
-    const endpointConfigChanged = useEndpointConfigStore_changed();
     const serverUpdateRequired = useEndpointConfig_serverUpdateRequired();
 
     // Resource Config Store
     const resourceConfig = useResourceConfig_resourceConfig();
+    const resourceConfigErrorsExist =
+        useResourceConfig_resourceConfigErrorsExist();
 
     const endpointConfigErrorFlag = editWorkflow
-        ? endpointConfigChanged() && endpointConfigErrorsExist
+        ? endpointConfigErrorsExist
```

Comment:
Looking at the logic, I cannot see a scenario where the encrypted endpoint config in the store could differ from that on the server (which is the evaluation performed by the `endpointConfigChanged` function) that should affect the generate button. `endpointConfigChanged` appropriately influences whether the data loss modal appears if you attempt to navigate away from the page, but I do not think it should factor in here. Thoughts?

Reply:
Sounds right to me. I was trying to test this same chunk of code in the OAuth branch and could never get this portion to work correctly.